commands/xinfo-consumers.md

```diff
@@ -4,7 +4,8 @@ The following information is provided for each consumer in the group:
 
 * **name**: the consumer's name
 * **pending**: the number of pending messages for the client, which are messages that were delivered but are yet to be acknowledged
-* **idle**: the number of milliseconds that have passed since the consumer last interacted with the server
+* **idle**: the number of milliseconds that have passed since the consumer last interacted with the server (Example: `XREADGROUP`)
+* **inactive**: the number of milliseconds that have passed since the consumer last successfully interacted with the server (Example: `XREADGROUP` that actually read some entries into the PEL)
```

Comment:
i'm not sure if `PEL` is a term we can use here is it extensively used in the rest of the streams documentation?

Reply:
It is, unfortunately/thanks god it is used throughout the docs.

@guy where does `XACK` and `X[AUTO]CLAIM` fit? Please add to the "examples"

Reply:
added

for the record, XACK is out of the game because , oddly, it's not consumer-specific